### ISSUE TITLE
Windows XP SP3 - regression in HttpRequest.dll between 1.0.5 and 1.0.6

### ISSUE THREAD

**Issue by karabaja4** [wqweto/VbAsyncSocket#51]
Hello,

running this VBSscript:
```
Const WinHttpRequestOption_SslErrorIgnoreFlags = 4
Const SslErrorFlag_UnknownCA = 256

With CreateObject("HttpRequest.cHttpRequest")
    .Open_ "GET", "https://www.howsmyssl.com/a/check"
    .Option_(WinHttpRequestOption_SslErrorIgnoreFlags) = SslErrorFlag_UnknownCA
    .Send
    WScript.echo Replace(.ResponseText, ",", vbCrLf)
End With
```

on Windows XP SP3 with `HttpRequest.dll` from the release `1.0.6` gives this error message:

![image](https://github.com/user-attachments/assets/2965f241-675f-4086-95da-9c67127471a8)

However, running it with `HttpRequest.dll` from the release `1.0.5` works OK:

![image](https://github.com/user-attachments/assets/1b808de7-e0af-49bb-afd1-d9c324910b80)

`HttpRequest.dll` is registered system-wide using `regsvr32`.

Regards,

Igor

**Comment by wqweto** [comment]
Thanks for the bug-report!

This issue with XP is fixed in commit [f91a052](https://github.com/wqweto/VbAsyncSocket/commit/f91a05203831cc070ef864ca02844178f7cb38d4).

Also ZIP files with binaries updated in [1.0.6 release](https://github.com/wqweto/VbAsyncSocket/releases/tag/HRR-1.0.6)

**Comment by karabaja4** [comment]
It works, thank you!
